internal/controller/nodeop_controller.go

```diff
@@ -274,11 +274,6 @@ func (r *NodeOpReconciler) drainNode(ctx context.Context, node *corev1.Node, dra
 			continue
 		}
 
-		// Skip pods that are part of the kube-system namespace
-		if pod.Namespace == "kube-system" {
-			continue
-		}
```

Comment:
Let them drain. We had `traefik` and other Pods that we didn't let be scheduled on other Nodes. No reason to do that.